layers/+spacemacs/spacemacs-visual/funcs.el

```diff
@@ -25,7 +25,7 @@
 ;; ansi-colors
 
 (defun spacemacs-visual//compilation-buffer-apply-ansi-colors ()
-  (when (memq 'compilation-mode (parent-mode-list major-mode))
+  (when (memq 'compilation-mode (get major-mode 'derived-mode-parent))
```

Comment:
Should this be `(when (derived-mode-p 'compilation-mode) ...)` to account for multiple levels of derivation?

Reply:
great. please submit PR.